ledger/distribution/src/main.rs

```diff
@@ -105,38 +104,41 @@ impl S3BlockWriter {
     }
 
     fn write_bytes_to_s3(&self, path: &str, filename: &str, value: &[u8]) {
-        let result: Result<
-            retry::OperationResult<(), ()>,
-            retry::Error<retry::OperationResult<(), RusotoError<PutObjectError>>>,
-        > = retry::retry(
-            retry::delay::Exponential::from_millis(10).map(retry::delay::jitter),
+        use futures::executor::block_on;
```

Comment:
Nit: Any reason why these are here and not at the top of the file?

Reply:
Moved.